server/src/utils/mixerConnections/SSLMixerConnection.ts

```diff
@@ -57,6 +57,119 @@ export class SSLMixerConnection {
         return str.substring(1)
     }
 
+    handleReceivedFaderLevelCommand = (buffer: any) => {
+        try {
+            let channelIndex = buffer[6]
+            let value = buffer.readUInt16BE(7) / 1024

```

Comment:
`value` is a very nondescript variable name. What is it the value of? Looking at the surrounding code my guess would be it is the fader level, so why not call it `faderLevel`? I shouldn't have to guess, since guesses leaves room for guessing wrong